src/App.php

```diff
@@ -36,23 +36,22 @@ class App extends AbstractApp {
 	 * @param \Slim\Slim $slim Application
 	 */
 	protected function configureSlim( \Slim\Slim $slim ) {
-		$slim->config(
-			[ 'displayErrorDetails' => true,
-			  'debug' => true,
-			  'oauth.enable' => Config::getBool( 'USE_OAUTH', false ),
-			  'oauth.consumer_token' => Config::getStr( 'OAUTH_CONSUMER_TOKEN' ),
-			  'oauth.secret_token' => Config::getStr( 'OAUTH_SECRET_TOKEN' ),
-			  'oauth.endpoint' => Config::getStr( 'OAUTH_ENDPOINT' ),
-			  'oauth.redir' => Config::getStr( 'OAUTH_REDIR' ),
-			  'oauth.callback' => Config::getStr( 'OAUTH_CALLBACK' ),
-			  'db.dsnwp' => Config::getStr( 'DB_DSN_WIKIPROJECT' ),
-			  'db.dsnen' => Config::getStr( 'DB_DSN_ENWIKI' ),
-			  'db.dsnpl' => Config::getStr( 'DB_DSN_PLAGIABOT' ),
-			  'db.user' => Config::getStr( 'DB_USER' ),
-			  'db.pass' => Config::getStr( 'DB_PASS' ),
-			  'templates.path' => '../public_html/templates'
-			]
-		);
+		$slim->config( [
+			'displayErrorDetails' => true,
+			'debug' => true,
+			'oauth.enable' => Config::getBool( 'USE_OAUTH', false ),
+			'oauth.consumer_token' => Config::getStr( 'OAUTH_CONSUMER_TOKEN' ),
+			'oauth.secret_token' => Config::getStr( 'OAUTH_SECRET_TOKEN' ),
+			'oauth.endpoint' => Config::getStr( 'OAUTH_ENDPOINT' ),
+			'oauth.redir' => Config::getStr( 'OAUTH_REDIR' ),
+			'oauth.callback' => Config::getStr( 'OAUTH_CALLBACK' ),
+			'db.dsnwp' => Config::getStr( 'DB_DSN_WIKIPROJECT' ),
+			'db.dsnen' => Config::getStr( 'DB_DSN_ENWIKI' ),
+			'db.dsnpl' => Config::getStr( 'DB_DSN_PLAGIABOT' ),
+			'db.user' => Config::getStr( 'DB_USER' ),
+			'db.pass' => Config::getStr( 'DB_PASS' ),
+			'templates.path' => '../public_html/templates'
+		] );
```

Comment:
matching indentation / bracket style with how we do our functions, hopefully this is OK